google-cloud-build/ci-scripts/scripts/decrypt.sh

```diff
@@ -9,4 +9,4 @@ KEY_NAME=$4
 mkdir -p secrets
 echo -n "$ENCRYPTED_SECRET" |
 	base64 -d -w 0 |
-       	gcloud kms decrypt --plaintext-file=- --ciphertext-file=- --location=global --keyring=$KEY_RING_NAME --key=$KEY_NAME > "secrets/$SECRET_NAME"
+       	gcloud kms decrypt --plaintext-file=- --ciphertext-file=- --location=global --keyring="$KEY_RING_NAME" --key="$KEY_NAME" > "secrets/$SECRET_NAME"
```

Comment:
hm, here we might need to check the documentation gcloud, if the way we quote is correct here, because it might be `"--keyring=$KEY_RING_NAME"` instead

Reply:
Done